modules/dnn/test/test_precomp.hpp

```diff
@@ -95,22 +66,6 @@ static testing::internal::ParamGenerator<Target> availableDnnTargets()
     return testing::ValuesIn(targets);
 }
 
-static testing::internal::ParamGenerator<tuple<Backend, Target> > dnnBackendsAndTargets()
```

Comment:
Moved into test_common.hpp (file is shared between accuracy and perf tests)

Reply:
@alalek, Could you please also replace `DNNBackend` and `DNNTarget` at https://github.com/opencv/opencv/blob/3.4/modules/dnn/perf/perf_net.cpp?

Reply:
Done, these changes are placed as a separate commit.